backend/ravegenieApp/models/publisher.py

```diff
@@ -60,7 +60,13 @@ def has_campaign_of_type(self, campaign_type):
 			if sub.is_active() and sub.campaign_type == campaign_type:
 				return True
 		return False
-							
+
+	@property
```

Comment:
If you're going to make it a property then adding the get to the  method name isn't necessary. Secondly, getter shouldn't return a Boolean if the value is not present. It's good practice to return None or an empty user at worst.